RuddockWebsite/__init__.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, g
+from sqlalchemy import create_engine
+
+from RuddockWebsite import config, constants
+from RuddockWebsite.modules import admin, hassle, users
+
+app = Flask(__name__)
+app.debug = False
```

Comment:
Do we want to specify debug mode here or in `app.run()`? Best to not do it in both places.